docs/client.rst

```diff
@@ -12,131 +12,109 @@ Let's start accessing gridded data, i.e., data that is stored as a regular multi
 
 .. doctest::
 
-    >>> print sst.shape
+    >>> print(sst.shape)
     (12, 90, 180)
     >>> grid = sst[0,10:14,10:14]  # this will download data from the server
-    >>> print grid
-    <class 'pydap.model.GridType'>
-        with data
-    [[ -1.26285708e+00  -9.99999979e+33  -9.99999979e+33  -9.99999979e+33]
-     [ -7.69166648e-01  -7.79999971e-01  -6.75454497e-01  -5.95714271e-01]
-     [  1.28333330e-01  -5.00000156e-02  -6.36363626e-02  -1.41666666e-01]
-     [  6.38000011e-01   8.95384610e-01   7.21666634e-01   8.10000002e-01]]
-        and axes
-    366.0
-    [-69. -67. -65. -63.]
-    [ 41.  43.  45.  47.]
+    >>> print(grid)
+    <GridType with array 'SST' and maps 'TIME', 'COADSY', 'COADSX'>
 
 The data itself can be accessed in the ``array`` attribute of the Grid, and also on the individual axes:
 
 .. doctest::
 
-    >>> print grid.array[:]
-    [[ -1.26285708e+00  -9.99999979e+33  -9.99999979e+33  -9.99999979e+33]
-     [ -7.69166648e-01  -7.79999971e-01  -6.75454497e-01  -5.95714271e-01]
-     [  1.28333330e-01  -5.00000156e-02  -6.36363626e-02  -1.41666666e-01]
-     [  6.38000011e-01   8.95384610e-01   7.21666634e-01   8.10000002e-01]]
-    >>> print grid.COADSX[:]
+    >>> print(grid.array[:])
+    <BaseType with data array([[[ -1.26285708e+00,  -9.99999979e+33,  -9.99999979e+33,
+              -9.99999979e+33],
+            [ -7.69166648e-01,  -7.79999971e-01,  -6.75454497e-01,
+              -5.95714271e-01],
+            [  1.28333330e-01,  -5.00000156e-02,  -6.36363626e-02,
+              -1.41666666e-01],
+            [  6.38000011e-01,   8.95384610e-01,   7.21666634e-01,
+               8.10000002e-01]]], dtype=float32)>
+    >>> print(grid.array[:].data)
+    [[[ -1.26285708e+00  -9.99999979e+33  -9.99999979e+33  -9.99999979e+33]
+      [ -7.69166648e-01  -7.79999971e-01  -6.75454497e-01  -5.95714271e-01]
+      [  1.28333330e-01  -5.00000156e-02  -6.36363626e-02  -1.41666666e-01]
+      [  6.38000011e-01   8.95384610e-01   7.21666634e-01   8.10000002e-01]]]
+    >>> print(grid.COADSX[:])
+    <BaseType with data array([ 41.,  43.,  45.,  47.])>
+    >>> print(grid.COADSX[:].data)
     [ 41.  43.  45.  47.]
 
 Alternatively, we could have dowloaded the data directly, skipping the axes:
 
 .. doctest::
 
-    >>> print sst.array[0,10:14,10:14] 
-    [[ -1.26285708e+00  -9.99999979e+33  -9.99999979e+33  -9.99999979e+33]
-     [ -7.69166648e-01  -7.79999971e-01  -6.75454497e-01  -5.95714271e-01]
-     [  1.28333330e-01  -5.00000156e-02  -6.36363626e-02  -1.41666666e-01]
-     [  6.38000011e-01   8.95384610e-01   7.21666634e-01   8.10000002e-01]]
-
-Instead of indexes we can also subset the data using its maps, in a more natural way. Just keep in mind that sometimes axes can be cyclic, like longitude, and you may have to download two separate parts and concatenate them together. This is not the case here:
```

Comment:
Unfortunately, it does not work anymore. It might be possible to reintroduce without too much pain but at the moment ``GridType`` does not support array slicing :(